ClosedXML/Excel/Cells/XLCell.cs

```diff
@@ -867,19 +866,17 @@ void resetRecordPosition()
             else
             {
                 const BindingFlags bindingFlags = BindingFlags.Public | BindingFlags.Instance | BindingFlags.Static;
-                var memberCache = new Dictionary<Type, IEnumerable<MemberInfo>>();
-                var accessorCache = new Dictionary<Type, TypeAccessor>();
-                IEnumerable<MemberInfo> members = null;
-                TypeAccessor accessor = null;
+                var memberCache = new Dictionary<Type, MemberInfo[]>();
+                MemberInfo[] members = null;
                 bool isPlainObject = itemType == typeof(object);
 
                 if (!isPlainObject)
                 {
                     members = itemType.GetFields(bindingFlags).Cast<MemberInfo>()
                          .Concat(itemType.GetProperties(bindingFlags))
                          .Where(mi => !XLColumnAttribute.IgnoreMember(mi))
-                         .OrderBy(mi => XLColumnAttribute.GetOrder(mi));
-                    accessor = TypeAccessor.Create(itemType);
+                         .OrderBy(mi => XLColumnAttribute.GetOrder(mi))
+                         .ToArray();
```

Comment:
Do you know of any Visual Studio plugins that can warn me about silly enumerator mistakes like these?

Reply:
Resharper indicates possible multiple enumerations of `IEnumerable` (https://www.jetbrains.com/help/resharper/2020.1/PossibleMultipleEnumeration.html). This may help.